### DISCUSSION TITLE
Can't get the editor to start, and I don't know if it's a bug or if doing something wrong.

### DISCUSSION THREAD

**Discussion by nicalpi** [Milkdown/milkdown#711] [Q&A]
Hi,

I'm trying to get somewhere and boot the edit in a Rails app, but somehow I can't even make the simplest thing work.

My code is like this (here it's using Stimulus, but I've tried in a straight JS file and it's the same result)

```javascript
import { Controller } from "@hotwired/stimulus"
import { Ctx, rootCtx, defaultValueCtx, Editor, editorViewCtx, editorViewOptionsCtx, themeManagerCtx } from '@milkdown/core';

export default class extends Controller {
 async connect() {
    this.editor = Editor.make()
      .config((ctx) => {
        ctx.set(rootCtx, this.element);
      })

    this.editor.create();
  }

  disconnect() {
  }
}
```

But, when the controller connects, I get this error in the console

```
emotion-css-create-instance.esm.js:23 Uncaught (in promise) TypeError: (0 , import_cache.default) is not a function
    at se (emotion-css-create-instance.esm.js:23:15)
    at we (emotion.ts:7:60)
    at theme.ts:38:25
    at async Promise.all (index 0)
    at async Q2.create (editor.ts:107:9)
```

Given how many people do have the editor working, I'm sure I'm the one doing something wrong. Could anyone be able to help please?

**Comment by Saul-Mirone** [comment]
Did you add any theme and preset?

**Reply by nicalpi** [reply]
Nope. The code I have here is the only code that is doing something from milkdown.

I have tried to strip things as much as I can to get to the bottom of that issue.

**Reply by nicalpi** [reply]
PS when I had the theme nord and commonmark (like the basic get started from the main website) I have the same issue.

**Reply by Saul-Mirone** [reply]
I think it's related to your build toolchain. Maybe you can try [vite ruby](https://vite-ruby.netlify.app/guide/rails.html)?

**Comment by z-950** [comment]
I think you can try clear npm cache and install again.

**Reply by nicalpi** [reply]
Nope, I've tried.

Unfortunately, I think the issue if in `@emotion/css`

At the top there's an import like `import createCache from '@emotion/cache';`

I think there's an issue with esbuild and non scoped import that is causing the issue.
